Fixed Clock Issue and Refactored Luminous Refresh Rate

Dariensg opened the pull request:
-Grabbed Biome from the registry key in clock mixin since getResourceLocation was returning null
-Added extra checks to make sure that there shouldn't be a NullPointerException
-Refactored Luminous Refresh Rate to Luminous Refresh Delay for clarity
-Updated lang files to match this refactor

Dariensg committed:
Fixed Clock Issue and Refactored Luminous Refresh Rate

-Grabbed Biome from the registry key in clock mixin since getResourceLocation was returning null
-Added extra checks to make sure that there shouldn't be a NullPointerException
-Refactored Luminous Refresh Rate to Luminous Refresh Delay for clarity
-Updated lang files to match this refactor

SwanX1 commented on src/main/resources/infernal-expansion.mixins.json:
```diff
@@ -29,6 +29,7 @@
     "client.MixinPaintingSpriteUploader",
-    "client.MixinPlayerRenderer"
+    "client.MixinPlayerRenderer",
+    "client.MixinItemModelsProperties"
```
Mixins should be listed alphabetically.

SwanX1 commented on src/main/java/com/nekomaster1000/infernalexp/mixin/client/MixinItemModelsProperties.java:
```diff
@@ -1,28 +1,38 @@
-		if (world.getBiome(entity.getPosition()).getRegistryName().equals(IEBiomes.GLOWSTONE_CANYON.getId())) {
-			return MathHelper.nextDouble(new Random(), 0.95, 1.05) % 1;
-		}
+        ClientWorld clientWorld = world;
+        if (entity == null) {
+            return in;
+        }
+        if (world == null && entity.world instanceof ClientWorld) {
+	        clientWorld = (ClientWorld)entity.getEntityWorld();
+        }
+
+        Optional<RegistryKey<Biome>> biome = clientWorld.func_242406_i(entity.getPosition());
+
+	    if (biome.isPresent() && biome.get().getLocation().equals(IEBiomes.GLOWSTONE_CANYON.getId())) {
+	        return MathHelper.nextDouble(new Random(), 0.95, 1.05) % 1;
+        }
```
please use consistent indentaton

SwanX1 reviewed (changes requested):
Please a

Dariensg committed:
Requested Fixes

-Hopefully the indentation is consistent because it doesn't have a visual difference in my IDE
-Fixed alphabetical ordering of mixins
